cmake/gen/avx2_microkernels.cmake

```diff
@@ -419,6 +419,10 @@ SET(ALL_AVX2_MICROKERNEL_SRCS
   src/math/gen/f32-tanh-avx2-expm1minus-rr1-p6h5ts-div.c
   src/math/gen/f32-tanh-avx2-expm1minus-rr1-p6h5ts-nr1.c
   src/math/gen/f32-tanh-avx2-expm1minus-rr1-p6h5ts-nr1adj.c
+  src/qd8-f16-qb4w-gemm/gen/qd8-f16-qb4w-gemm-1x8c8-minmax-avx2.c
```

Comment:
I think you forgot to add the c2 microkernel here. When i ran the update-microkernels.py it added it in for me.

Reply:
I've removed the c2 microkernel. I did not intend for it to be in the PR, sorry about that. It was a warmup that I handwrote.